packages/devtools_test/README.md

```diff
@@ -1,6 +1,6 @@
 ## What is this?
 
-This is a package containing test utilities for use when writing Dart DevTools [Dart DevTools](https://pub.dev/packages/devtools) tests.
+This is a package containing test utilities for use when writing [Dart DevTools](https://pub.dev/packages/devtools) tests.
```

Comment:
can you replace the link with https://docs.flutter.dev/development/tools/devtools/overview?

We should remove any lingering references to pub as we no longer ship there.